autophonetreeherder.py

```diff
@@ -191,13 +191,23 @@ def _create_job(self, tjc, machine, build_url, project, revision, build_type, bu
         # Fake the buildername from buildbot...
         tj.add_artifact('buildapi', 'json', {
             'buildername': buildername(machine_platform, project, build_type, t.name)})
-        # Create a 'privatebuild' artifact for storing information
-        # regarding the build.
-        tj.add_artifact('privatebuild', 'json', {
+        # Add job details for storing information regarding the build (so we can
+        # retrigger them)
+        # also, for backwards compatibility, store an artifact with the same
+        # information (can remove this soon)
+        privatebuild = {
             'build_url': build_url,
             'config_file': t.config_file,
             'chunk': t.chunk,
-            'builder_type': builder_type})
+            'builder_type': builder_type
+        }
+        tj.add_artifact('Job Info', 'json', {
+            'job_details': [
+                {'title': k, 'value': v } for
```

Comment:
C:206, 0: No space allowed before bracket
                {'title': k, 'value': v } for
                                        ^ (bad-whitespace)